packages/lesswrong/components/search/UsersSearchAutocompleteHit.tsx

```diff
@@ -0,0 +1,47 @@
+import {Components, registerComponent} from '../../lib/vulcan-lib'
+import React from 'react'
+import {styles as metaInfo} from '../common/MetaInfo'
+
+const specificityFmClass = 'forum-magnum'
+const specificityCkClass = 'ck-override'
+const styles = (theme: ThemeType): JssStyles => ({
+  userHitLabel: {
+    // A specificity hack to work around https://github.com/ckeditor/ckeditor5/issues/3424
+    [`&.${specificityFmClass}.${specificityCkClass}, &.${specificityFmClass}.${specificityCkClass} *`]: {
+      ...theme.typography.body2,
+      ...metaInfo(theme).root,
+      marginLeft: theme.spacing.unit,
+    },
+  },
+})
+
+
+interface UsersSearchAutocompleteHitProps {
+  classes: ClassesType
+  displayName: string
+  createdAt: Date
+  karma?: number
+}
+
+const UsersSearchAutocompleteHit = (props: UsersSearchAutocompleteHitProps) => {
+  const {MetaInfo, FormatDate} = Components
+
+  const metaClassName = `${props.classes.userHitLabel} ${specificityFmClass} ${specificityCkClass} ck-override`
+  return <span>
+    {props.displayName}
```

Comment:
This should use `userGetDisplayName`, from users/helpers.tsx. (This is pretty non-obvious, at some point we should figure out a good eslint setup that warns when you use certain fields that are actually a bit of a footgun)

Reply:
Will update! curious what is the footgun around that field? is it missing sometimes/something else?

Reply:
ok, looking at it seems alignment forum has a different notation, which probably implies that I should retrieve `fullName` from search above as-well